packages/neuron-wallet/tests/controllers/multisig.test.ts

```diff
@@ -77,13 +77,13 @@ const multisigConfig = {
     params: {
       multisig_configs: {
         sighash_addresses: multisigBlake160s.map(args =>
-          scriptToAddress(
+          helpers.encodeToAddress(
             {
               args,
-              codeHash: systemScripts.SECP256K1_BLAKE160.codeHash,
-              hashType: systemScripts.SECP256K1_BLAKE160.hashType,
+              codeHash: config.predefined.AGGRON4.SCRIPTS.SECP256K1_BLAKE160.CODE_HASH,
```

Comment:
For this part, we are just using `SECP256K1_BLAKE160` code hash, which is the same in mainnet/testnet/devnet. 

Because `lumos` utils like `helpers.encodeToAddress`, `parseAddress` needs to receive a config, so I plan to create a static method to provide config like `NeuronInstance.getLumosConfig()`